cmd/format-fs.go

```diff
@@ -230,6 +242,32 @@ func initFormatFS(ctx context.Context, fsPath string) (rlk *lock.RLockedFile, er
 		if err != nil {
 			return nil, err
 		}
+
+		format := &formatFSV2{}
```

Comment:
I think we should do this after formatFS migration (in the code below)

Reply:
Adding a separate function that will be called initially, before anything else is done.